js/initcontext.go

```diff
@@ -238,7 +240,8 @@ func (i *InitContext) Open(ctx context.Context, filename string, args ...string)
 	}
 
 	if len(args) > 0 && args[0] == "b" {
-		return i.runtime.ToValue(data), nil
+		b := i.runtime.NewArrayBuffer(data)
```

Comment:
hmm while returning the JS equivalent to `[]byte` from `open()` is not ideal, it was somewhat usable. So I am not sure if we should break it, since some scripts probably depend on the current strange behavior. Why not just add another flag value for an `ArrayBuffer` response?

Reply:
Yeah, I also wasn't sure if we should break it.

> Why not just add another flag value for an `ArrayBuffer` response?

:-/ Sure, maybe `ab`?

Reply:
Done, see c593025f.